api/controllers/api-user.js

```diff
@@ -17,10 +19,42 @@ const get = (request, response) => {
 }
 
 const post = (request, response) => {
-  const requestBody = request.body
-  ApiUser.save(requestBody).then(() => {
-    response.json(requestBody)
-  })
+  let requestBody = request.body
+
+  // lookup guid
+  if (!requestBody.guid && requestBody.contact_email) {
+    const req = https.get('https://thekey.me/cas/api/' + process.env.THE_KEY_API_KEY + '/user/attributes?email=' + requestBody.contact_email, (res) => {
```

Comment:
`THE_KEY_API_KEY` needs to be added to serverless/environment.js so it gets configured in lambda as well as values added to blackbox for staging and production.

Reply:
It was added to blackbox yesterday